src/three-components/SmoothControls.ts

```diff
@@ -304,6 +308,11 @@ export class SmoothControls extends EventDispatcher {
     this[$spherical].copy(this[$targetSpherical]);
   }
 
+  updateZoom(framedHeight: number) {
```

Comment:
Separately, should we use a more generalized concept than `framedHeight` for `SmoothControls`? Our project is the only consumer of this module today, but for the most part the module should work in other, non-`<model-viewer>` Three.js contexts..

Reply:
I’m not convinced this framing system is awesome, but I was trying to keep my changes minimal. Perhaps with a little design work we could make this a follow on work item?

Reply:
I think renaming this method to `updateFramedHeight` is an easy thing we could do right now.
Also, it would be good to add a comment regardless. `zoom` seems to be an underdocumented concept right now.

If you can file an issue to bookmark that we haven't formally established the relationship between "framed height" and `SmoothControls`, that's fine for now. I would also leave a reference to that issue in a comment here.